backend/tournesol/serializers/subsample.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+from rest_framework.fields import IntegerField
+
+from tournesol.serializers.entity import RelatedEntitySerializer
+from tournesol.serializers.poll import CollectiveRatingSerializer
+from tournesol.serializers.rating import ExtendedInvididualRatingSerializer
+
+
+class SubSampleMetadataSerializer(serializers.Serializer):
+    bucket = IntegerField(read_only=True)
```

Comment:
This is not very important, would it just be in order the numbers 0, 1, 2, 3, 4, etc.. ?

If yes, we don't need to include this in the result

Reply:
In our case, when the `offset` parameter is provided, the first bucket returned will not necessarily be 1.